packages/site/src/lib/helpers/entry/displayFields.test.ts

```diff
@@ -0,0 +1,61 @@
+import type { IEntry } from '@living-dictionaries/types';
+import {
+  showLocalOrthographies,
+  showPartsOfSpeech,
+  removeItalicTagsWithAPeriod,
+  unnecessaryLineBreaksRegex,
+} from './displayFields';
+import { entriesWithAlternateOrthographies } from '../../../routes/[dictionaryId]/entries/print/mock-data';
+
+describe('showLocalOrthographies', () => {
+  const mockEntries: IEntry[] = entriesWithAlternateOrthographies;
+
+  test('shows some local orthographies', () => {
+    expect(showLocalOrthographies(mockEntries[1]).join(', ')).toMatchInlineSnapshot(
+      '"さよなら, 안녕, αντίο"'
+    );
+  });
+  test('shows all local orthographies', () => {
+    expect(showLocalOrthographies(mockEntries[0]).join(', ')).toMatchInlineSnapshot(
+      '"Nnọọ, Привет, سلام, नमस्ते, שלום"'
+    );
+  });
+});
+
+describe('showPartsOfSpeech', () => {
+  const partsOfSpeech = {
+    stringPOS: 'n',
+    arrayPOS1: ['n', 'adj'],
+    arrayPOS2: ['v'],
+    emptyArray: [],
+    emptyString: '',
+    null: null,
+  };
+
+  test('Shows parts of speech', () => {
+    expect(showPartsOfSpeech(partsOfSpeech.stringPOS)).toMatchInlineSnapshot('"n."');
```

Comment:
@jacob-8 I'm doing multiple expects here since I think they're very related and the partsOfSpeech object keys are self explanatory. But if you think I need to make a test for each one, I can change it.

Reply:
To answer that question we need to take a step back. A goal of tests is that tests should be **readable**. When I look at these tests, they don't tell me anything further than what the function names already say: 
<img width="235" alt="image" src="https://user-images.githubusercontent.com/7559478/215625036-27c1da42-c7be-4959-8839-1448a7aba349.png">

One reason to use multiple test blocks is to document functionality and then to know precisely which piece of functionality is broken. So yes, we definitely need multiple tests here. A test should just test one thing, and this one is doing 6.

Reply:
Also note that the describe block is not a hard requirement if you only have one test. The reason we use it is that we often have more than one test per function and it adds clarity as to what is broken. I've split out the tests and updated the expected result for one of them as it was incorrect. You didn't make the latest changes, but I can imagine that you can pretty quickly pick this back up and work to fix it: 
<img width="1841" alt="image" src="https://user-images.githubusercontent.com/7559478/215626629-5694908c-98bf-4e15-99a9-074bc92e3519.png">

Reply:
Sometimes it's helpful to create a constant to pass to a test, but in this case it makes it longer to write, and harder to read. For something as simple as a one letter string or `null` just write it inline.